reference-client/src/main/java/com/transformuk/hee/tis/reference/client/ReferenceService.java

```diff
@@ -84,6 +84,14 @@
    */
   List<GradeDTO> findGradesByName(String name);
 
+  /**
+   * Find the grades with status=CURRENT and placementGrade=true and trainingGrade=true.
```

Comment:
Really useful doc.  I would have assumed it was "placementGrade=true **OR** trainingGrade=true".

Reply:
I know, its a bit weird, but them's the business rules :). I think the whole piece of work might be temporary (not that that would be an excuse for bad code: you know how many 'temporary' things end up being quite permanent!).  In theory it will be refactored again when we get to https://hee-tis.atlassian.net/browse/TIS21-371